libs/libcommon/src/libcommon/parquet_utils.py

```diff
@@ -248,17 +249,17 @@ def query_truncated_binary(self, offset: int, length: int) -> tuple[pa.Table, li
                 pq.ParquetFile(
```

Comment:
pq.ParquetFile can accept a `filesystem` argument so I think it could be further simplified and the mocking in the tests could be simpler by passing a local filesystem rather than a network filesystem but leaving it as is since hopefully we can remove it entirely.